app/src/forms/rbac/routes.js

```diff
@@ -52,4 +52,8 @@ routes.delete('/users', hasFormPermissions([P.TEAM_UPDATE]), hasFormRoles([R.OWN
   await controller.removeMultiUsers(req, res, next);
 });
 
+routes.get('/form/user', hasFormPermissions([P.SUBMISSION_CREATE, P.SUBMISSION_DELETE, P.SUBMISSION_READ, P.SUBMISSION_REVIEW, P.SUBMISSION_REVIEW]), async (req, res, next) => {
```

Comment:
Not sure you want all those permissions... A submitter who is not a reviewer can make this call.
Could go with `hasFormPermissions([P.FORM_READ])`, this function is read-only so I think it's ok just to check they have form access... makes this more easily re-used.

Reply:
fixed